src/workers/notifications/postAdded.ts

```diff
@@ -42,6 +42,10 @@ const worker: NotificationWorker = {
       return;
     }
 
+    if (post.flags?.vordr) {
```

Comment:
Missing test for this?

Reply:
It's here
https://github.com/dailydotdev/daily-api/blob/AS-463-vordr-posts/__tests__/workers/notifications/postAdded.ts

Reply:
Nice, missed it in diff 🙏